Sources/Apodini/Configurations/Schedule.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import class Vapor.Application
+import SwifCron
+
+/// `Configuration` to start `Job`s at server startup.
+public class Schedule<K: ApodiniKeys, T: Job>: Configuration {
+    private let scheduler = Scheduler.shared
+    private let job: T
+    private let cronTrigger: String
+    private let runs: Int?
+    private let keyPath: KeyPath<K, T>
+    
+    /// Initializes the `Schedule` configuration.
+    ///
+    /// - Parameters:
+    ///     - job: The background running task conforming to `Job`s.
+    ///     - on: Crontab as a String.
+    ///     - keyPath: Associates a `Job` for later retrieval.
+    public init(_ job: T, on cronTrigger: String, _ keyPath: KeyPath<K, T>) {
+        self.job = job
+        self.cronTrigger = cronTrigger
+        self.runs = nil
+        self.keyPath = keyPath
+    }
+    
+    /// Initializes the `Schedule` configuration.
+    ///
+    /// - Parameters:
+    ///     - job: The background running task conforming to `Job`s.
+    ///     - on: Crontab as a String.
+    ///     - runs: Number of times a `Job` should run.
+    ///     - keyPath: Associates a `Job` for later retrieval.
+    public init(_ job: T, on cronTrigger: String, runs: Int, _ keyPath: KeyPath<K, T>) {
```

Comment:
You could use default parameter values and save the other initializer:

```suggestion
    public init(_ job: T, on cronTrigger: String, runs: Int? = nil, _ keyPath: KeyPath<K, T>)
```

Reply:
I prefer to have to separate initializers because I think if you want to use `runs` than it doesn’t make sense to declare it as `nil`.

Reply:
Makes sense 👍